packages/sigint-collector/src/simulation/SignalGenerator.js

```
@@ -0,0 +1,302 @@
+"use strict";
+/**
+ * Signal Generator - Creates simulated signals for training
+ * TRAINING/SIMULATION ONLY
+ */
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.SignalGenerator = void 0;
+const uuid_1 = require("uuid");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)